client/notifications/index.jsx

```diff
@@ -123,6 +124,15 @@ var Notifications = React.createClass({
 		} );
 	},
 
+	postAuth: function() {
+		if ( config.isEnabled( 'oauth' ) && oAuthToken.getToken() !== false ) {
```

Comment:
It'd be nice to assign `oAuthToken.getToken()` to a variable and re-use since it parses cookies every time you call it.